docker/docker-compose.yml

```diff
@@ -83,55 +83,58 @@ services:
-      aws --endpoint-url=http://localstack:4575 sns list-subscriptions
+      - /bin/sh
+      - -c
+      - |
+        sleep 30
+        aws --version
+        aws --endpoint-url=http://localstack:4572 s3 mb s3://untrusted-bucket
+        aws --endpoint-url=http://localstack:4572 s3 mb s3://trusted-bucket
+        aws --endpoint-url=http://localstack:4576 sqs create-queue --queue-name document-queue
+        aws --endpoint-url=http://localstack:4576 sqs create-queue --queue-name document-queue-dlq
+        aws --endpoint-url=http://localstack:4576 sqs create-queue --queue-name audit-queue
+        aws --endpoint-url=http://localstack:4576 sqs create-queue --queue-name audit-queue-dlq
+        aws --endpoint-url=http://localstack:4576 sqs create-queue --queue-name search-queue
+        aws --endpoint-url=http://localstack:4576 sqs create-queue --queue-name search-queue-dlq
+        aws --endpoint-url=http://localstack:4576 sqs create-queue --queue-name notify-queue
+        aws --endpoint-url=http://localstack:4576 sqs create-queue --queue-name notify-queue-dlq
+        aws --endpoint-url=http://localstack:4575 sns create-topic --name hocs-audit-topic
+        aws --endpoint-url=http://localstack:4575 sns subscribe --topic-arn arn:aws:sns:eu-west-2:123456789012:hocs-audit-topic --attributes RawMessageDelivery=true --protocol sqs --notification-endpoint arn:aws:sns:eu-west-2:123456789012:audit-queue
+        aws --endpoint-url=http://localstack:4575 sns subscribe --topic-arn arn:aws:sns:eu-west-2:123456789012:hocs-audit-topic --attributes RawMessageDelivery=true --protocol sqs --notification-endpoint arn:aws:sns:eu-west-2:123456789012:search-queue
+        aws --endpoint-url=http://localstack:4575 sns set-subscription-attributes --subscription-arn $$(aws --endpoint-url=http://localstack:4575 sns list-subscriptions-by-topic --topic-arn arn:aws:sns:eu-west-2:123456789012:hocs-audit-topic --output json | jq --raw-output '.Subscriptions[0].SubscriptionArn') --attribute-name RawMessageDelivery --attribute-value true
+        aws --endpoint-url=http://localstack:4575 sns set-subscription-attributes --subscription-arn $$(aws --endpoint-url=http://localstack:4575 sns list-subscriptions-by-topic --topic-arn arn:aws:sns:eu-west-2:123456789012:hocs-audit-topic --output json | jq --raw-output '.Subscriptions[1].SubscriptionArn') --attribute-name RawMessageDelivery --attribute-value true
+        aws --endpoint-url=http://localstack:4575 sns list-subscriptions
+        aws --endpoint-url=http://localstack:4576 sqs create-queue --queue-name ukvi-complaint-queue
+        aws --endpoint-url=http://localstack:4576 sqs create-queue --queue-name ukvi-complaint-queue-dlq
+        sleep 8h
```

Comment:
yikes. If it makes you feel better init containers landed in docker-compose trunk a few days ago

Reply:
I can't find any docs for that. I'll look again at the next change.